### PR TITLE
Update in-development.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the document to reflect that On-demand proactive remediation isn't in Public Preview

fixes#https://github.com/MicrosoftDocs/memdocs-pr/issues/13335

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/fundamentals/in-development.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ Applies to:
 For more information about configuring Settings Catalog profiles in Intune, see [Create a policy using settings catalog](../configuration/settings-catalog.md).
 
 ### On-demand proactive remediation for a Windows device<!-- 14783338  -->  
-A new device action that is in public preview allows you to run a proactive remediation on-demand to a single Windows device. The **Run** remediation device action will allow you to resolve issues without having to wait for a proactive remediation to run on its assigned schedule. You will also be able to view the status of proactive remediations under **Remediations** in the **Monitor** section of a device.
+A new device action that is in Private preview allows you to run a proactive remediation on-demand to a single Windows device. The **Run** remediation device action will allow you to resolve issues without having to wait for a proactive remediation to run on its assigned schedule. You will also be able to view the status of proactive remediations under **Remediations** in the **Monitor** section of a device.
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/in-development.md:134]:
```suggestion
A new device action that is in private preview allows you to run a proactive remediation on-demand to a single Windows device. The **Run** remediation device action will allow you to resolve issues without having to wait for a proactive remediation to run on its assigned schedule. You will also be able to view the status of proactive remediations under **Remediations** in the **Monitor** section of a device.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dougeby Please signoff and merge this PR. Thanks

**Comment by Brenduns** [comment]
Closing without other action.  This PR has long been irrelivent. Subsequent to creation, the feature involved released to public preview making this PR a non-issue.

**Pull request closed without merging**